newIDE/app/src/CommandPalette/CommandHooks.js

```diff
@@ -50,3 +80,27 @@ export const useKeyboardShortcutForCommandPalette = (onOpen: () => void) => {
     [onOpen, values.useCommandPalette]
   );
 };
+
+/**
+ * React component for using useCommand hook in
+ * class components
+ */
+export const UseCommandHook = (props: {
+  commandName: string,
+  command: SimpleCommand,
+}) => {
+  useCommand(props.commandName, props.command);
+  return null;
+};
+
+/**
+ * React component for using useCommandWithOptions
+ * hook in class components
+ */
+export const UseCommandWithOptionsHook = (props: {
+  commandName: string,
+  command: CommandWithOptions<*>,
```

Comment:
* is deprecated (see https://flow.org/en/docs/linting/rule-reference/#toc-deprecated-type), I recommend here either:
* typing generically the function component: `... = <OptionType>(props: { commandName: string, command: CommandWithOptions<OptionType>}) => { ... };`
* or just use any